### PR TITLE
new package: iptables 1.8.4

### PR DESCRIPTION
**Pull request by a-schaefers**
## Checklist

- [x] Latest upstream version.
- [x] I agree to maintain this package (**required**).
- [x] I agree to notify KISS if I can no longer maintain this package (**required**).

### CODE REVIEW

File: community/iptables/build [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/sh -e
+
+# give option to easily enable nftables by passing USE="--with-nftables" to kiss
+[ -z "$USE" ] && USE="--disable-nftables"
```
Comment (dylanaraps) [inline community/iptables/build]: KISS has no USE flags. The user can simply fork the package into their own repository.

NOTE: Symlinks to the KISS repositories work for unchanged files. 

Example:

```
- build (modified to enable nftables)
- checksums -> /path/to/community/iptables/checksums
- sources   -> /path/to/community/iptables/sources
- version   -> /path/to/community/iptables/version
```

This removes the burden of maintaining the modified package yourself. :)

File: community/iptables/build [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+autoreconf -i
+
+./configure --prefix=/usr \
+            --sbindir=/usr/sbin \
+            --enable-libipq --with-xtlibdir=/lib/xtables "$USE"
```
Comment (dylanaraps) [inline community/iptables/build]: Expand this to:

```sh
./configure \
    --prefix=/usr \
    --bla-bla \
    --bla-bla
```

File: community/iptables/build [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+make
+
```
Comment (dylanaraps) [inline community/iptables/build]: Remove this blank line.

File: community/iptables/depends [hunk 4]
```diff
@@ -0,0 +1 @@
+make
```
Comment (dylanaraps) [inline community/iptables/depends]: Depends file can be removed.

File: community/iptables/build (hunk 3 above)
Comment (a-schaefers) [reply community/iptables/build]: look again, i was already fixing before you commented

File: community/iptables/build (hunk 1 above)
Comment (a-schaefers) [reply community/iptables/build]: agreed

File: community/iptables/build (hunk 3 above)
Comment (dylanaraps) [reply community/iptables/build]: Apologies. :+1:

Comment (a-schaefers) [reply community/iptables/build]: Try again, i'm confused which blank line is troublesome

Comment (dylanaraps) [reply community/iptables/build]: The blank line between `make` and `make install`.

File: community/emacs/build [hunk 5]
```diff
@@ -5,4 +5,5 @@
     --with-gnutls=no
 
 make
+
 make DESTDIR="$1" install
```
Comment (dylanaraps) [inline community/emacs/build]: This file is unrelated to the PR. :+1:

Comment (a-schaefers) [reply community/emacs/build]: lol WAT is going on here

File: community/iptables/build (hunk 3 above)
Comment (a-schaefers) [reply community/iptables/build]: okay got it  thanks, was worried because Emacs throws an extra line EOF unless I change a setting.

**Comment by a-schaefers** [comment]
ready

File: community/iptables/build [hunk 6]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh -e
+
+./configure --prefix=/usr \
```
Comment (dylanaraps) [inline community/iptables/build]: Add a newline before `--prefix=/usr`.

File: community/iptables/sources [hunk 7]
```diff
@@ -0,0 +1 @@
+http://www.netfilter.org/projects/iptables/files/iptables-1.8.4.tar.bz2
```
Comment (dylanaraps) [inline community/iptables/sources]: Change this to: `https://www.netfilter.org/projects/iptables/files/iptables-1.8.4.tar.bz2` (https)

Comment (a-schaefers) [reply community/iptables/sources]: nice

File: community/iptables/build (hunk 6 above)
Comment (dylanaraps) [reply community/iptables/build]: Indentation overall may need adjusting after this change. :+1:
